packages/catalog-client/src/CatalogClient.ts

```diff
@@ -418,7 +418,7 @@ export class CatalogClient implements CatalogApi {
     }
   }
 
-  private async requestRequired<T = any>(response: Response): Promise<T> {
+  private async requestRequired<T>(response: TypedResponse<T>): Promise<T> {
```

Comment:
should we also update the other methods in here too like `requestOptional` etc? Just to see if there's any other typescript gremlins?

Reply:
Aye there was one that was unnecessarily optional actually, but I'll ship this one since that's effectively harmless, to get the patch out

Reply:
Following up in a bit